src/server/acl/user_registry.cc

```diff
@@ -14,21 +14,23 @@
 
 ABSL_DECLARE_FLAG(std::string, requirepass);
 
+using namespace util;
```

Comment:
I am really not a big fun of using namespaces because it pulls everything and can be potentially dangerous. Maybe in this context is fine but I am making the comment for completeness. I am *not* strongly advocating this though.